packages/components/src/components/files/FileTree.tsx

```diff
@@ -220,8 +221,9 @@ export class FileTree extends PureComponent<FileTreeProps, FileTreeState> {
                         }];
                     }
 
+                    const rootName = defaultRootName ? defaultRootName : 'root';
```

Comment:
You can replace this line by putting the default value for this prop up in the `static defaultProps` for this component.

Reply:
Done!